examples/lib/mapHtmltypes.js

```diff
@@ -14,6 +14,9 @@ function mapHtmlTypes(inputInfo){
   case 'select':
     htmlType = 'select';
     break;
+  case 'button':
```

Comment:
We don't want a button type. The output type should still be string.

Reply:
Done